### PR TITLE
[New] Sensitive Registry Hive Access via RegBack

### PR DESCRIPTION
**Pull request by Samirbous**
https://x.com/Defte_/status/1807743567210729910

image

### CODE REVIEW

File: rules/windows/credential_access_regback_sam_security_hives.toml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    "Use Case: Threat Detection",
+    "Tactic: Credential Access",
+    "Resources: Investigation Guide",
+    "Data Source: Elastic Defend",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.action == "open" and event.outcome == "success" and process.executbale != null and
```
Comment (w0rk3r) [inline rules/windows/credential_access_regback_sam_security_hives.toml:75]:
```suggestion
file where event.action == "open" and event.outcome == "success" and process.executable != null and 
```

File: rules/windows/credential_access_regback_sam_security_hives.toml [hunk 2]
```diff
@@ -0,0 +1,105 @@
+[metadata]
+creation_date = "2024/07/01"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/07/01"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies attempts to access sensitive registry hives which contain credentials from the registry backdup folder."
```
Comment (w0rk3r) [inline rules/windows/credential_access_regback_sam_security_hives.toml:9]:
```suggestion
description = "Identifies attempts to access sensitive registry hives which contain credentials from the registry backup folder."
```

File: rules/windows/credential_access_regback_sam_security_hives.toml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+risk_score = 73
+rule_id = "63e381a6-0ffe-4afb-9a26-72a59ad16d7b"
+setup = """## Setup
+
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2,
+events will not define `event.ingested` and default fallback for EQL rules was not added until version 8.2.
+Hence for this rule to work effectively, users will need to add a custom ingest pipeline to populate
+`event.ingested` to @timestamp.
+For more details on adding a custom ingest pipeline refer - https://www.elastic.co/guide/en/fleet/current/data-streams-pipeline-tutorial.html
+"""
```
Comment (w0rk3r) [inline rules/windows/credential_access_regback_sam_security_hives.toml:61]:
```suggestion
```

File: rules/windows/credential_access_regback_sam_security_hives.toml [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.action == "open" and event.outcome == "success" and process.executable != null and 
+ file.path :
+      ("?:\\Windows\\System32\\config\\RegBack\\SAM",
+       "?:\\Windows\\System32\\config\\RegBack\\SECURITY",
+       "?:\\Windows\\System32\\config\\RegBack\\SYSTEM") and 
+ not (user.id == "S-1-5-18" and process.executbale : "?:\\Windows\\system32\\taskhostw.exe")
```
Comment (w0rk3r) [inline rules/windows/credential_access_regback_sam_security_hives.toml:72]:
```suggestion
 not (user.id == "S-1-5-18" and process.executable : "?:\\Windows\\system32\\taskhostw.exe")
```

**Review by terrancedejesus — APPROVED** [review]
Nice addition ✅

**Review by Aegrah — APPROVED** [review]

**Pull request merged**